views/config_notifications.mako

```diff
@@ -1565,7 +1565,12 @@
                                         <span class="component-desc">
                                             <select id="trakt_default_indexer" name="trakt_default_indexer" class="form-control input-sm">
                                                 % for indexer in indexerApi().indexers:
-                                                <option value="${indexer}" ${'selected="selected"' if app.TRAKT_DEFAULT_INDEXER == indexer else ''}>${indexerApi().indexers[indexer]}</option>
+                                                    <%
+                                                        name = indexerApi().indexers[indexer]
```

Comment:
You are calling the class every time to do this.
Instead save indexerApi().indexers to a variable and use that in the loop.